python/pylibcugraph/pylibcugraph/heterogeneous_biased_neighbor_sample.pyx

```diff
@@ -0,0 +1,424 @@
+
+    edge_biases: FIXE: update this - Create edge_biases of type
+        'cugraph_edge_property_view_t' - edge biases not yet supported.
+
+    start_vertex_list: device array type
+        Device array containing the list of starting vertices for sampling.
+
+    start_vertex_offsets: list[int] (Optional)
+        Offsets of each label within the start vertex list.
+
+    h_fan_out: tuple of numpy array type
+        Device array containing the branching out (fan-out) degrees per
+        starting vertex for each hop level
+
+        The sampling method can use different fan_out values for each edge type
+        which is not the case for homogeneous neighborhood sampling.
+
+    num_edge_types: Number of edge types where a value of 1 translates to homogeneous neighbor
+        sample whereas a value greater than 1 translates to heterogeneous neighbor
+        sample.
+
+    with_replacement: bool
+        If true, sampling procedure is done with replacement (the same vertex
+        can be selected multiple times in the same step).
+
+    do_expensive_check: bool
+        If True, performs more extensive tests on the inputs to ensure
+        validitity, at the expense of increased run time.
+
+    with_edge_properties: bool
+        If True, returns the edge properties of each edges along with the
+        edges themselves.  Will result in an error if the provided graph
+        does not have edge properties.
+
+    prior_sources_behavior: str (Optional)
+        Options are "carryover", and "exclude".
+        Default will leave the source list as-is.
+        Carryover will carry over sources from previous hops to the
+        current hop.
+        Exclude will exclude sources from previous hops from reappearing
+        as sources in future hops.
+
+    deduplicate_sources: bool (Optional)
+        If True, will deduplicate the source list before sampling.
+        Defaults to False.
+
+    renumber: bool (Optional)
+        If True, will renumber the sources and destinations on a
+        per-batch basis and return the renumber map and batch offsets
+        in additional to the standard returns.
+
+    retain_seeds: bool (Optional)
+        If True, will retain the original seeds (original source vertices)
+        in the output even if they do not have outgoing neighbors.
+        Defaults to False.
+
+    compression: str (Optional)
+        Options: COO (default), CSR, CSC, DCSR, DCSR
+        Sets the compression format for the returned samples.
+
+    compress_per_hop: bool (Optional)
+        If False (default), will create a compressed edgelist for the
+        entire batch.
+        If True, will create a separate compressed edgelist per hop within
+        a batch.
+
+    random_state: int (Optional)
+        Random state to use when generating samples.  Optional argument,
+        defaults to a hash of process id, time, and hostname.
+        (See pylibcugraph.random.CuGraphRandomState)
+
+    return_dict: bool (Optional)
```

Comment:
Since hetergeneous_biased_neighbor_sample() is a new API, why not have this default to True so that when deprecated users won't see a difference from the default?